examples/JavaProntoClient/src/main/webapp/allCatalogs.jsp

```diff
@@ -0,0 +1,32 @@
+<%@ page language="java" contentType="text/html; charset=UTF-8"
+    pageEncoding="UTF-8"%>
+<%@taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c"%>
+
+<!DOCTYPE html>
+<html>
+	<link rel="stylesheet" type="text/css" href="style.css">
```

Comment:
Maybe it's just git, but the indentation on this line doesn't line up with the next line. I think it might be a mixing of tabs and spaces? The indentation inside the body also seems off now (the h1 on line 10 appears to be indented less than the h2 on line 11)